cc/gtp_player.cc

```diff
@@ -338,21 +331,21 @@ GtpPlayer::Response GtpPlayer::HandleKnownCommand(CmdArgs args) {
   return Response::Ok(result);
 }
 
-GtpPlayer::Response GtpPlayer::HandleKomi(CmdArgs args) {
+GtpClient::Response GtpClient::HandleKomi(CmdArgs args) {
   auto response = CheckArgsExact(1, args);
   if (!response.ok) {
     return response;
   }
 
   double x;
-  if (!absl::SimpleAtod(args[0], &x) || x != game()->options().komi) {
+  if (!absl::SimpleAtod(args[0], &x) || x != player_->game()->options().komi) {
```

Comment:
this probably works but is it guaranteed that

<DOUBLELITERAL> == SimpleAtod("<DOUBLELITERAL>")

e.g. 
1.3f == SimpleAtod("1.3")

Reply:
"normal" komi values of X.5 are exactly representable in IEEE floating point format, so this should be ok for all cases we care about.